packages/utils/src/fecha/index.ts

```
@@ -0,0 +1,342 @@
+    parseFlags[name] = parseFlags[name.substr(0, 1).toLowerCase()]
+  }
+})
+
+fecha.format = (dateObj, mask, i18nSettings) => {
+  const i18n = i18nSettings || fecha.i18n
+
+  if (typeof dateObj === 'number') {
+    dateObj = new Date(dateObj)
+  }
+
+  if (!isDate(dateObj) || isNaN(dateObj.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)